setup.py

```diff
@@ -9,7 +9,7 @@
 
 setup(
     name='django-ses-plus',
-    version='0.2.2',
+    version='0.2.3',
```

Comment:
Should we bump this version to `0.3.0` or `1.0.0`? There isn't any backward incompatible changes but we are using this library for more than 1 year as far as I can see.

Reply:
I think we can start using version 1.